scripts/run_graphite_web.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+cd $(git rev-parse --show-toplevel)
+source scripts/env.sh
+
+GRAPHITE_WEB_CONF="${BG_VENV}/lib/python2.7/site-packages/graphite/local_settings.py"
```

Comment:
`python2.7` depends on the default python binary, which might be python 3

Reply:
Done.
(but unsure that I did it in a correct way)